### PR TITLE
Ft/data privacy

### PR DESCRIPTION
**Pull request by beylar**
Data privacy information

### CODE REVIEW

**Review by josueahadi — APPROVED** [review]
The data privacy page looks perfect

**Review by Ntwalii — APPROVED** [review]

File: src/sections/greenheader.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    </div>
+    <img id="button"
+      class="self-end md:w-[9%] lg:hidden"
+      src="./src/images/green-menu.svg"
+      alt="menu-bar"
+    />
+  </div>
+  
+  
+
```
Comment (Kingcedru) [inline src/sections/greenheader.html]: Remove these spaces

File: src/sections/greenheader.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<div class="flex justify-between items-center px-7 py-3">
+    <div class="flex flex-col text-sm md:text-2xl lg:flex-1 self-start">
+      <p>DE</p>
+      <p>EN</p>
+    </div>
+    <div class="relative h-10">
+       <img class="md:hidden" src="./src/images/white.svg">
```
Comment (Kingcedru) [inline src/sections/greenheader.html]: You missed the alt attribute on this image tag

File: src/sections/header.html [hunk 3]
```diff
@@ -1,13 +1,23 @@
...
+    <a href="./contact.html"><p>Contact</p></a>
+  </div>
+  <img id="button"
+    class="self-end md:w-[9%] lg:hidden"
+    src="./src/images/menu.svg"
+    alt="menu-bar"
+  />
 </div>
+
+
```
Comment (Kingcedru) [inline src/sections/header.html]: Remove spaces

File: src/sections/header.html [hunk 4]
```diff
@@ -1,13 +1,23 @@
...
-        <a href="#services"><p>Services</p></a>
-        <a href="#references"><p>References</p></a>
-        <a href="./contact.html"><p>Contact</p></a>
-    </div>
+  <div class="flex flex-col text-sm md:text-2xl lg:flex-1 self-start">
+    <p>DE</p>
+    <p>EN</p>
+  </div>
+  <div class="relative h-10">
+     <img class="md:hidden" src="./src/images/white.svg">
```
Comment (Kingcedru) [inline src/sections/header.html]: Also add the image description.

**Review by Kingcedru — CHANGES_REQUESTED** [review]
Changes needed.

**Review by josueahadi — APPROVED** [review]

File: privacy.html
Comment (josueahadi) [inline privacy.html:1]: Excellent! All that is left is adding the recent changes from the footer hotfix branch to this page's footer, and this page will be complete.

**Review by josueahadi — CHANGES_REQUESTED** [review]

**Review by josueahadi — APPROVED** [review]

**Review by NoellaPromise — APPROVED** [review]

**Review by Kingcedru — APPROVED** [review]

**Pull request merged**